packages/grafana-ui/src/components/uPlot/config.ts

```diff
@@ -70,16 +70,16 @@ export interface LineConfig {
 /**
  * @alpha
  */
-export interface AreaConfig {
+export interface FillConfig {
```

Comment:
This change could be its own PR... it touches a bunch of files, but the shared config felt awkward with "Area" rather than "Fill"